core/tests/test_views.py

```diff
@@ -117,12 +150,12 @@ def test_add_comment_unauthorized(client: Client, post: Post) -> None:
     assert reverse("login") in response.url
 
 
-def test_add_nested_comment_valid(client: Client, another_user: User, post: Post, comment: Post) -> None:
```

Comment:
Why change here? If not needed please revert.  I don't see need for this change.